app/Http/Controllers/StatusController.php

```diff
@@ -352,4 +359,17 @@ public static function getStatusesByEvent(?string $slug, ?int $id): array {
 
         return ['event' => $event, 'statuses' => $statuses->simplePaginate(15)];
     }
+
+    public static function getFutureCheckins(): Paginator {
+        return auth()->user()->statuses()
+                     ->with('user',
+                            'trainCheckin',
+                            'trainCheckin.Origin',
+                            'trainCheckin.Destination',
+                            'trainCheckin.HafasTrip',
+                            'event')->orderBy('created_at', 'DESC')
+                     ->whereHas('trainCheckin', function($query) {
+                         $query->where('departure', '>=', date('Y-m-d H:i:s', strtotime("+20min")));
+                     })->simplePaginate(15);
```

Comment:
Sure you want to paginate future checkins and not just show them all? It's in a dropdown anyway and a user should not have so many.